live_pipeline/transform/transform_functions.py

```diff
@@ -0,0 +1,170 @@
+"""libraries required for transformation functions"""
+from datetime import datetime as dt
+from os import environ
+from boto3 import client
+from dotenv import load_dotenv
+import json
```

Comment:
`json` is not an external library and should come before `boto3` as per pep8 requirements https://peps.python.org/pep-0008/#imports